apps/src/music/views/MusicLabView.tsx

```diff
@@ -149,11 +151,13 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
 
   // Update loop that runs while playback is in progress.
   const doPlaybackUpdate = useCallback(() => {
+    checkForEnd();
```

Comment:
Minor: might be a bit simpler to do this in a `useEffect` rather than in the playback update loop, since we have most of the information we need in redux already (lastMeasure, currentPlayheadPosition, isPlaying)

Reply:
Good call.  Updated.